.github/workflows/scripts/check-files.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+set -eu
+
+SIZE_LIMIT=150000
+
+check_size() {
+	size="$(stat --printf="%s" "$1")"
+	if [ "$size" -gt "$SIZE_LIMIT" ]; then
+		echo "File $1 is bigger than specified $SIZE_LIMIT limit"
+		exit 1
+	fi
+}
+
+check_webp_name() {
```

Comment:
The only difference between this func and func below is the regular expression used. I think both functions could be unified taking regex as a parameter

Reply:
Fair enough. In fact, I think the way it is written now doesn't work if the file is in the wrong folder either, as it just prepends the folder to the filename without fixing the folder it is in.

So my latest update actually just removes that regex altogether